sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -793,15 +780,16 @@ public static void endSession() {
    * @param name the transaction name
    * @param operation the operation
    * @param description the description
-   * @param bindToScope if transaction should be bound to scope
+   * @param transactionOptions options for the transaction
    * @return created transaction
    */
   public static @NotNull ITransaction startTransaction(
       final @NotNull String name,
       final @NotNull String operation,
       final @Nullable String description,
```

Comment:
Do we wanna keep this overload that allows passing in a `description`?

Reply:
yep, I think I'd keep it, as description is quite popular to set, and we also use it for indexing spans in starfish, so it's an important field